src/Integration.UnitTests/Persistence/SolutionBindingSerializerTests.cs

```diff
@@ -78,7 +79,14 @@ public void TestInitialize()
         [TestMethod]
         public void SolutionBindingSerializer_ArgChecks()
         {
+            ILogger loggerMock = new Mock<ILogger>().Object;
+            IFile fileWrapper = new FileWrapper();
             Exceptions.Expect<ArgumentNullException>(() => new SolutionBindingSerializer(null));
+            Exceptions.Expect<ArgumentNullException>(() => new SolutionBindingSerializer(null, sourceControlledFileSystem, store, loggerMock, fileWrapper));
+            Exceptions.Expect<ArgumentNullException>(() => new SolutionBindingSerializer(serviceProvider, null, store, loggerMock, fileWrapper));
+            Exceptions.Expect<ArgumentNullException>(() => new SolutionBindingSerializer(serviceProvider, sourceControlledFileSystem, null, loggerMock, fileWrapper));
+            Exceptions.Expect<ArgumentNullException>(() => new SolutionBindingSerializer(serviceProvider, sourceControlledFileSystem, store, null, fileWrapper));
+            Exceptions.Expect<ArgumentNullException>(() => new SolutionBindingSerializer(serviceProvider, sourceControlledFileSystem, store, loggerMock, null));
```

Comment:
I re-added these invalid argument tests here since the base class now throws rather than asserting.